solutions/usaco-784.mdx

```diff
@@ -59,9 +61,45 @@ int main(){
 	cout << maxCover;
 	return 0;
 }
-
 ```
 
 </CPPSection>
+</LanguageSection>
+
+## Python Implementation
+
+**Time Complexity:** $\mathcal{O}(N^2*\text{TIME})$
```

Comment:
`\text{TIME}` in the time complexity is quite visually unpleasant. Prefer just saying "$\mathcal{O}(N^2*T)$, where $T$ is the maximum time" or something (just be consistent with above)